src/ceph_mon.cc

```
@@ -296,6 +296,8 @@ int main(int argc, const char **argv)
       inject_monmap = val;
     } else if (ceph_argparse_witharg(args, i, &val, "--extract-monmap", (char*)NULL)) {
       extract_monmap = val;
+    } else if (ceph_argparse_flag(args, i, "--no-config-file", (char*)NULL)) {
+      continue;
```

this silently ignores the option?

---

actually, forgot to remove this.